packages/web-components/src/text-field/fixtures/text-field.html

```diff
@@ -93,3 +93,16 @@ <h2>In a form</h2>
   <fluent-text-field name="fname" aria-label="Text field with aria-label"></fluent-text-field>
   <input type="submit" value="submit" />
 </form>
+
+<!-- With Fabric Core Icon -->
+<h4>With Fabric Core Icon in start</h4>
+<fluent-text-field>
+  Label
+  <i slot="start" class="ms-Icon ms-Icon--TestPlan"></i>
+</fluent-text-field>
+
+<h4>With Fabric Core Icon in end</h4>
+<fluent-text-field>
+  Label
+  <i slot="end" class="ms-Icon ms-Icon--TestPlan"></i>
+</fluent-text-field>
```

Comment:
Similar to the above, while others may still be using it, our current POR is to leverage the Fluent icons which likely requires a little bit more discussion around how to best consume and leverage these. If we can remove this as part of this commit it would be appreciated.